project.go

```diff
@@ -56,6 +56,9 @@ type Project struct {
 // ProjectListOptions represents the options for listing projects
 type ProjectListOptions struct {
 	ListOptions
+
+	// Optional: String (partial project name) used to filter the results.
```

Comment:
Can you mention that multiple names can be matched using comma-separated values?
```suggestion
	// Optional: String (partial project name) used to filter the results. If multiple, comma separated values are specified, projects matching any of the names are returned.
```

Reply:
Done!